packages/beacon-node/src/network/core/networkCore.ts

```diff
@@ -197,9 +196,7 @@ export class NetworkCore implements INetworkCore {
 
     const enr = opts.discv5?.enr;
     const nodeId = enr ? fromHexString(ENR.decodeTxt(enr).nodeId) : null;
-    const attnetsService = opts.deterministicLongLivedAttnets
-      ? new DLLAttnetsService(config, clock, gossip, metadata, logger, metrics, nodeId, opts)
-      : new AttnetsService(config, clock, gossip, metadata, logger, metrics, opts);
+    const attnetsService = new DLLAttnetsService(config, clock, gossip, metadata, logger, metrics, nodeId, opts);
```

Comment:
same here, we can keep the naming but as DLL subnets are the de factor standard now, could consider removing prefix